apps/dashboard/app/models/script.rb

```diff
@@ -211,9 +211,7 @@ def write_job_options_to_cache(opts)
   end
 
   def cache_file_path
-    Pathname.new(project_dir).join(".ondemand/scripts/#{id}_opts.json")
-    # TODO: fix bug 
-    #Pathname.new("#{Script.scripts_dir(project_dir)}/#{id}_cache.json")
+    Pathname.new("#{Script.scripts_dir(project_dir)}/#{id}_cache.json")
```

Comment:
tests may fail because you changed this name but I think `#{id}_cache.json` is better.

Reply:
It would help if I read my own comment about that too :man_facepalming: I'll change that.